include/tvm/runtime/device_api.h

```diff
@@ -46,10 +46,10 @@ enum DeviceAttrKind : int {
 };
 
 /*! \brief Number of bytes each allocation must align to */
-constexpr int kAllocAlignment = 64;
+constexpr int kAllocAlignment = 128;
 
 /*! \brief Number of bytes each allocation must align to in temporary allocation */
-constexpr int kTempAllocaAlignment = 64;
+constexpr int kTempAllocaAlignment = 128;
```

Comment:
How about this? https://github.com/dmlc/tvm/blob/master/src/pass/ir_util.h#L174 I think we should handle it specially for Hexagon.

Reply:
Someone suggested having a target-specific function that gets the required alignment, instead of having global constants.  I think that would be a better solution.  This patch simply has what we've done about it.
I haven't looked into it yet, is there a precedent for implementing such a thing?  Do you have any suggestions as to how it should be done?

Reply:
I also suggest we have a target-specific function that gets the required alignment. Then in  `GetTempAllocaAlignment` we could get this alignment and just simply return it for Hexagon (Need detect the target is Hexagon, for example we define the macro `USE_HEXAGON` in CMakeLists.txt).

Reply:
Same comment. How about create target-specific function alignment?